app/helpers/application_helper.rb

```diff
@@ -500,7 +500,12 @@ def current_url_params(permitted: [])
   end
 
   def app_metadata
-    { UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url }
+    {
+      UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url,
+      location: Location.current.attributes.slice('id', 'title'),
+      organization: Organization.current.attributes.slice('id', 'title'),
```

Comment:
Needs a check if current loc/org is set, otherwise fails on undefined method when in 'Any Context'.

Reply:
Thanks @xprazak2, should work now